packages/shared/src/utils/__test-sample.ts

```
@@ -0,0 +1,20 @@
+export function divide(a: number, b: number) {
+  return a / b;
+}
+
+export function parseAge(input: any): number {
```

ESLint

[error] Unexpected any. Specify a different type.

(`@typescript-eslint/no-explicit-any`)

---

[error] Unexpected any. Specify a different type.

(`@typescript-eslint/no-explicit-any`)

---

[error] Unexpected var, use let or const instead.

(no-var)